app/src/main/java/org/mozilla/reference/browser/components/Push.kt

```diff
@@ -17,6 +18,7 @@ import org.mozilla.reference.browser.push.FirebasePush
  * Component group for push services. These components use services that strongly depend on
  * push messaging (e.g. WebPush, SendTab).
  */
+@SuppressLint("DiscouragedApi")
```

Comment:
What was the error again that we were suppressing?

Reply:
On `val resId = context.resources.getIdentifier("project_id", "string", context.packageName)` it's saying:
> It is much more efficient to retrieve resources by identifier (e.g. R.foo.bar) than by name (e.g. getIdentifier("bar", "foo", null)).